client/extensions/woocommerce/state/sites/settings/stripe-connect-account/actions.js

```diff
@@ -10,8 +10,6 @@
 import {
 	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CLEAR_COMPLETED_NOTIFICATION,
 	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CLEAR_ERROR,
-	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE,
-	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_COMPLETE,
```

Comment:
We will probably need to remove these as well. https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/state/action-types.js#L209-L212